### DISCUSSION TITLE
I hope that astronomical calculations can calculate phenomena between moons and parent planets: eclipse, occultation, transit, and greatest elongation (angular distance from parent planet).

### DISCUSSION THREAD

**Discussion by sunshuwei** [Stellarium/stellarium#800] [Q&A] [answered]
It will help us observe planetary phenomena using telescopes (such as Jupiter and its moons).

**Comment by gzotti** [comment, marked as answer]
Please check discussion and links in #222

**Comment by alex-w** [comment]
AstroCalc can compute some phenomena for planetary moons,  but very limited.

**Comment by sunshuwei** [comment]
Thanks, it seems that the orbit accuracy is not enough or the shape is irregular..
If it is only the major moons of Jupiter and Saturn？
Other moons are difficult to observe for amateur astronomers.

**Comment by gzotti** [comment]
Don't completely understand. Do you want to simulate those (the ones discussed in #222 work great after Guillaume's changes), or do you need predictions for future events in the AstroCalc module? Probably there is dedicated software for such phenomena.
